omod/src/main/resources/apps/ugandaemr_reports_app.json

```diff
@@ -79,7 +79,16 @@
         "label": "Appointments List",
         "url": "reportingui/runReport.page?reportDefinition=9c85e20b-c3ce-4dc1-b332-13f1d02f1c5c",
         "order": 5
-      },{
+      },
```

Comment:
@solemabrothers These reports should be next to each other in the order that they are expected to be seen

Reply:
@ssmusoke did you mean Over Due and Due For Viral Load reports??